frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -781,14 +781,9 @@ export function getEnabledLayers(
   datasetConfiguration: DatasetConfiguration,
   options: {
     invert?: boolean;
-    onlyColorLayers: boolean;
-  } = {
-    onlyColorLayers: false,
-  },
```

Comment:
The only code lines where the `onlyColorLayers` property was used are removed in this pr. Therefore I also remove this option to not have unused code lines